lib/e2e/src/system.rs

```diff
@@ -56,3 +55,25 @@ pub fn provider() -> Provider {
         .expect("failed to parse RPC_URL string into a URL");
     ProviderBuilder::new().with_recommended_fillers().on_http(rpc_url)
 }
+
+/// Send `amount` eth to `address` in the nitro-tesnode.
+pub fn fund_account(address: Address, amount: &str) -> eyre::Result<()> {
```

Comment:
Where should it live?

Reply:
I would put it in [`lib/e2e/src/account.rs`](https://github.com/OpenZeppelin/rust-contracts-stylus/blob/main/lib/e2e/src/account.rs) as it introduces logic connected with accounts. But I can agree on leaving it here as it is.

Reply:
I chose to move it to this module, since it is completely coupled to `nitro-testnode` (which is our `system`) and funding an account is kind of like a god-like operation, but this is not a strong opinion.